### DISCUSSION TITLE
Custom flags does not load in tournament mode

### DISCUSSION THREAD

**Discussion by kuroni** [ppy/osu#15837] [Q&A] [answered]
Description: I have the `flags` folder in the correct destination described [here](https://osu.ppy.sh/wiki/en/osu%21tourney/osu%21lazer_tournament_client#flags), but Lazer does not load the custom flag.

![image](https://user-images.githubusercontent.com/13513478/143688300-f35ffac4-3aaf-49ab-86a4-3ea2d49d8b29.png)

![image](https://user-images.githubusercontent.com/13513478/143688305-7efabc80-690d-4aee-9db0-f0871665c7c6.png)

`videos` do load by the way. Client is built from master, OS is Arch Linux.

[Log](https://pastebin.com/f9TubTDJ)

**Comment by bdach** [comment, marked as answer]
the `flags` folder should be named `Flags` instead. capital-`F`.

that said it kinda sucks that the wiki has casing that is "wrong" on some OSes that care about filename casing. windows (which is what most people will be using) doesn't care about casing, from what the internet tells me.

**Reply by kuroni** [reply]
Nice, thank you very much!
